cloud/datasync/src/index.ts

```diff
@@ -0,0 +1,145 @@
+import * as Promise from 'bluebird';
+import * as Logger from 'bunyan';
+import * as Express from 'express';
+import SyncGlobalOptions from './syncOptions';
```

Comment:
oh.. my bad. It looks like some unmerged commits appeared in this PR. Let's drop them!